src/containers/add-resources/AddResources.constants.tsx

```diff
@@ -0,0 +1,67 @@
+import Typography from '@mui/material/Typography'
+
+import AppChip from '~/components/app-chip/AppChip'
+import IconExtensionWithTitle from '~/components/icon-extension-with-title/IconExtensionWithTitle'
+
+import { getFormattedDate } from '~/utils/helper-functions'
+import { resourcesData } from '~/containers/course-section/CourseSectionContainer.constants'
+
+import { styles } from '~/containers/add-resources/AddResources.styles'
+
+import {
+  AdditionalPropsInterface,
+  RemoveColumnRules,
+  SortEnum,
+  CourseResources
+} from '~/types'
+
+export const initialSort = { order: SortEnum.Desc, orderBy: 'updatedAt' }
+
+export const columns = [
+  {
+    label: 'myResourcesPage.lessons.title',
+    field: 'title',
+    calculatedCellValue: (item: CourseResources) => {
+      let icon
+      if ('content' in item) {
```

Comment:
Better to create a new const file for each resource as we did for AddAttachments and AddQuestions and pass columns and other variables as a props to AddResources